hazelcast-jet-distribution/src/bin-filemode-755/jet-cluster-admin

```diff
@@ -20,14 +21,14 @@ if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
     exit 0
 fi
 
-command -v curl >/dev/null 2>&1 || { \
```

Comment:
this change makes each call to print `/usr/bin/curl` to output

Reply:
do we need `-v` there? I think that's the part that prints the text?

Reply:
yes without `-v` it does not work at all. `command -v` outputs where the binary for that command is

Reply:
I reverted the change, I got confused that this call was actually invoking curl